### PR TITLE
chore(ui): remove store dependency for step form [TCTC-9179]

### PR DESCRIPTION
**Pull request by alice-sevin**
1. Export some constants and getters/actions to be able to move dataviewer component to another repository
2. Remove store dependency from stepform component to be able to use it with query builder or without it.

Now stepform can be used as a dumb component using props or plug to store

### CODE REVIEW

File: ui/src/components/stepforms/StepFormComponent.vue
Comment (davinov) [inline ui/src/components/stepforms/StepFormComponent.vue:1]: :+1:

File: ui/stories/StepForm.stories.ts
Comment (davinov) [inline ui/stories/StepForm.stories.ts:1]: Thanks for including the stories :+1:

File: ui/src/main.ts [hunk 1]
```diff
@@ -33,6 +44,7 @@ export {
   NewDateInput,
   DateRangeInput,
   Pagination,
+  StepFormComponent,
```
Comment (davinov) [inline ui/src/main.ts:47]: Maybe this could be called `StepForm` only?

**Review by davinov — APPROVED** [review]
Awesome! I've not read in details all the modifications to each step form, but the API of the StepForm component is clear. Thanks for this work :clap:

File: ui/src/main.ts (hunk 1 above)
Comment (alice-sevin) [reply ui/src/main.ts:47]: There is already a StepForm component (it's the base form component used by each steps :wink: )

**Pull request merged**